lib/sender.js

```diff
@@ -10,12 +10,14 @@ var http         = require('http'),
     request      = require('request'),
     requestSync  = require('sync-request'),
 
+    api          = require('./api'),
```

Comment:
is this library require in this class ?

Reply:
oh, I just found out that this **api**  is not being used in sender.js file. I'll removed this line.